app/src/hooks/useGetCounter.tsx

```diff
@@ -0,0 +1,28 @@
+import { useState, useEffect } from 'react'
+
+export default function useGetCounter(seconds: number, isCountingUp: boolean) {
+  const [counterContent, setCounterContent] = useState(isCountingUp ? 0 : seconds)
+  const [isPaused, setIsPaused] = useState(false)
+
+  const handlePause = () => {
+    setIsPaused((isPaused) => !isPaused)
+  }
+
+  useEffect(() => {
+    const timer = setInterval(() => {
+      if (isPaused) return
+
+      setCounterContent((prevCounter) => {
+        if (isCountingUp) {
+          return prevCounter < seconds ? prevCounter + 1 : seconds
+        }
+
+        return prevCounter > 0 ? prevCounter - 1 : 0
+      })
+    }, 1000)
```

Comment:
The counting interval (now 1000) should not be hardcoded, but it could have a default value of 1000 though.

We might want to refresh this 10 times per second, for example, instead of just 1.